### ISSUE TITLE
Use different emojis for failure and error

### ISSUE THREAD

**Issue by piranna** [nearform/node-test-github-reporter#65]
https://github.com/nearform/node-test-github-reporter/blob/b600d81324e1d083f95e185f1ad903fac27878f4/index.js#L99-L100

I would use 💀 (`:skull:`) for failure, since most of the cases a test fails is because it died.

**Comment by simoneb** [comment]
Thanks for reporting this. I would prefer to keep the emojis agnostic from any human connotation, but if you would like to send a PR to make them configurable, feel free to do so.

**Comment by piranna** [comment]
It's mostly about differenciate between failures and errors, because they are not the same, failures are errors in the tests themselves (they are broken) instead of the code they are testing, so 🏚️ (`:derelict_house:`) could also works.

**Comment by simoneb** [comment]
Yes agreed. I asked ChatGPT to come up with a list of emojis that could represent something that's an error, and it came up with this. Any preference?

❌ Cross Mark
🚫 No Entry Sign
⚠️ Warning Sign
🛑 Stop Sign
🚨 Police Car Light
🚫🔧 Wrench and No Entry Sign
🚩 Triangular Flag
🆘 SOS Button
💥 Collision Symbol
📛 Name Badge

**Comment by piranna** [comment]
Of these ones, 💥 (`:boom:`) I think is the best one for failures.

What prompt did you used?

**Comment by simoneb** [comment]
Let's go with that then. Mind sending a tiny PR for this? 

As for the prompt:

> show me emojis for something that's an error

**Comment by piranna** [comment]
> Mind sending a tiny PR for this?

You have it at https://github.com/nearform/node-test-github-reporter/pull/66
